notebook/README.md

```diff
@@ -0,0 +1,37 @@
+# notebook
+
+A simple example using a Jupyter notbook with Cog
+
+## Usage
+
+First, make sure you've got the [latest version of Cog](https://github.com/replicate/cog#install) installed.
+
+Build the image:
+
+```sh
+cog build
+```
+
+Run the Jupyter notebook server with Cog:
+
+```sh
+cog run -p 8888 --debug jupyter notebook --allow-root --ip=0.0.0.0
+```
+
+Copy the notebook URL to your browser. It should look something like this:
+
+```sh
+http://127.0.0.1:8888/?token=eedb5f511a60b179d1a1b2c6395f0f20c02a08124bae6896
+```
+
+Save any changes you make to your notebook, then export it as a Python script:
+
+```sh
+jupyter nbconvert --to script notebook.ipynb # creates notebook.py
```

Comment:
Should this be `my_notebook.ipynb`? If not, what's the distinction between this filename and the files included in the example?

Reply:
Yep, it should. I changed it to `my_notebook` in haste to avoid ambiguity with the `$ jupyter notebook` command, which has nothing to do with the local filename.

Reply:
a4db315be6862210d6d8d2d0719c94bd6d39e6c6